CHANGELOG.md

```diff
@@ -1,6 +1,11 @@
 # Changelog
 All notable changes to this project will be documented in this file
 
+## [2.0.0] - 2020-04-16
```

Comment:
nit: Out of date lol

Reply:
Bah. I knew that would be out of date _immediately_. Fixed.